app/spaces/[spaceid]/setting/roles/components/members/memberManagement.tsx

```
@@ -0,0 +1,259 @@
+  }, [openAddMemberDrawer]);
+
+  const getSortedMembers = useCallback(
+    (memberIds: string[]) => {
+      const membersWithRoles: string[] = [];
+      const membersWithoutRoles: string[] = [];
+      memberIds.forEach((memberId) => {
+        const existingMember = members.find(
+          (member) => member.userId.zucityProfile?.author?.id === memberId,
+        );
+
+        if (existingMember && existingMember.roleId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)